src/main/java/Duke.java

```diff
@@ -1,10 +1,168 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+// I am using the Google Java Style Guide https://google.github.io/styleguide/javaguide.html
```

Comment:
For this module, we are following this [java coding standard](https://se-education.org/guides/conventions/java/basic.html).